dev/com.ibm.ws.microprofile.config.1.3_fat/test-applications/serverXMLApp/src/com/ibm/ws/microprofile/config13/serverXML/web/ServerXMLServlet.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config13.serverXML.web;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+/**
+ *
+ */
+@WebServlet("/ServerXMLServlet")
+public class ServerXMLServlet extends FATServlet {
```

Comment:
I dislike the way these tests are sprawled across two classes with each test having a method in both. Can we avoid that?